packages/elements/src/components/ino-input-file/ino-input-file.tsx

```diff
@@ -50,6 +66,57 @@ export class InputFile implements ComponentInterface {
     files: object[];
```

Comment:
Instead of the object Type I think we could type it `File[]`

Reply:
✅